resources/migrations/019_add_schema_column_to_table.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: 19
+      author: camsaul
+      changes:
+        - addColumn:
+            tableName: metabase_table
+            columns:
+              - column:
+                  name: schema
+                  type: VARCHAR
```

Comment:
No, that one doesn't work on MySQL either which is why I created this PR https://github.com/metabase/metabase/pull/1434

I agree that we need to add a test to Circle that uses MySQL as the backing DB so that we can catch these things more easily.  I just noticed this late last night.

I think that ideally going forward we use `TEXT` as the column type for values which are truly meant to be unbounded strings such as JSON or really long descriptions.  Otherwise we should use `VARCHAR(n)`, which while it shouldn't be necessary to put in the limit, is the most compatible way to define normal variable length columns which we know don't need to be overly long.